ui/pages/settings/contact-list-tab/add-contact/add-contact.component.js

```diff
@@ -68,8 +68,10 @@ export default class AddContact extends PureComponent {
       isValidHexAddress(input, { mixedCaseUseChecksum: true });
     const validEnsAddress = isValidDomainName(input);
 
-    if (!IS_FLASK && !validEnsAddress && !valid) {
+    if (!validEnsAddress && !valid) {
```

Comment:
Do we not need it for FLASK?

Reply:
it doesn't matter if we are in FLASK or not, the validation is the same